### ISSUE TITLE
Subscribe to RSS feeds as well as hEntry feeds

### ISSUE THREAD

**Issue by npdoty** [aaronpk/Monocle#8]
I'm not sure everyone agrees with this model, but this is what I'm most looking for in an indie reader. I want to use the "subscribe" model for as many things as I can, whether the publisher is using RSS, or marks up their pages with hEntry and other microformats, or some other option (email newsletters?).

It would also make it easier for me to move towards an indie reader or reading sites via their microformatted posts if I could read them in the same place as my existing feeds.

**Comment by julien51** [comment]
I have the same "vision". The data formats should be irrelevant to the user and the reader should be smart enough to find whatever it's looking for: RSS, Atom, h-Atom, or even weirder things like maybe [fragments](http://blog.superfeedr.com/fragment-subscription/)?

Newsletter/emails would also be great. Maybe that means creating an email endpoint for each user in the reader?

**Comment by aaronpk** [comment]
I agree, the underlying data format shouldn't matter, h-entry, RSS, email, are all reasonable inputs. I like the idea of being able to subscribe to email newsletters from the reader! I'll see what I can do with that.
